src/App.css

```diff
@@ -1,5 +1,12 @@
-*{
+@import url('https://fonts.googleapis.com/css2?family=Nunito:wght@400;700&display=swap');
+
+* {
     padding: 0;
     margin: 0;
     box-sizing: border-box;
+}
+
+:root {
+    --main-color: rgba(13, 19, 33, 1);
+    --second-color: rgba(197, 216, 109, 1);
```

Comment:
We can add these colors to tailwind configuration through extend. This will enable us to use them in text colors or backgrounds like `text-secondary bg-secondary`